src/chips/nrf51822/gpio.rs

```diff
@@ -89,7 +196,33 @@ impl IndexMut<usize> for Port {
     }
 }
 
-pub static mut PA : Port = Port {
+impl Port {
+    // GPIOTE interrupt
+    pub fn handle_interrupt(&self) {
```

Comment:
OK, I can add a comment. But I don't think the SAM4L is documented at all in this way.